page-objects/pages/TASKListPage.js

```diff
@@ -16,13 +16,14 @@ const taskTitleContainer = '.assignment.card .title';
 const taskDescriptionContainer = '.assignment .text-muted.ckcontent';
 const taskContainer = '.homework li.card';
 const deleteTaskButtonInPopup = '.delete-modal button.btn-submit';
-const clickWithoutDueDate = "//*[text()='Task19']";
-const ungradedTask = "//div[text()='Task19']";
+const clickWithoutDueDate = "//div[@data-testid = 'taskTitle' and text() = 'Task19']";
+const ungradedTask = "//div[@data-testid = 'taskTitle' and text() = 'Task19']";
```

Comment:
Same here, same selector for a different variable...
If we want a specific tasks, then we should do it like that: const taskName = "[data-testid='taskName']"; and we define the taskName in the scenario.